src/backend/distributed/connection/connection_configuration.c

```diff
@@ -227,7 +238,7 @@ CheckConninfo(const char *conninfo, const char **whitelist,
  */
 void
 GetConnParams(ConnectionHashKey *key, char ***keywords, char ***values,
-			  MemoryContext context)
+			  Index *nonGlobalParamStart, MemoryContext context)
 {
 	/* make space for the port as a string: sign, 10 digits, NUL */
 	char *nodePortString = MemoryContextAlloc(context, 12 * sizeof(char *));
```

Comment:
Yeah, I also don't like it, but our "assign at declaration and don't interleave declarations and code" style rule meant I wrote it this way (note the big assignments up top are declarations, too).

Gonna leave it here, since I think the alternative would have too much unrelated code churn.